### ISSUE TITLE
camel-jbang instead of maven

### ISSUE THREAD

**Issue by mgubaidullin** [apache/camel-kameleon#37]
@davsclaus 
Current version of kameleon uses maven to generate output project.
It is quite resource demanding and quite slow.
I've been thinking about replace project generation with new camel-jbang export stuff.
Thoughts? Potential issues?

**Comment by davsclaus** [comment]
Yes this is a good idea, we innovate much faster there and can also make the project output more dynamic than a maven archetype do.

Then we can also allow to choose build tool: maven or gradle
And for kamelets then we can compute the correct set of dependencies as well, so its all ready to go.
